src/NServiceBus.Core/Pipeline/BehaviorContext.cs

```diff
@@ -7,7 +7,7 @@ namespace NServiceBus
     /// <summary>
     /// Provides base context for behavior context implementations.
     /// </summary>
-    public abstract class BehaviorContext : ContextBag, IBehaviorContext
+    abstract class BehaviorContext : ContextBag, IBehaviorContext
```

Comment:
i would default to "no" since we can trivially revert that decision in a minor if we are wrong